architecture/standards/0005-introduce-core-ui-architecture-module.md

```diff
@@ -1,4 +1,4 @@
-# ADR-0004 - Introduce a UI architecture module to the core platform
```

Comment:
💭 Simple numbering mistakes like this are annoying and embarrassing.  Reserving a number for usage seems like it should be a "solved problem".

I think we should establish an "ADRs Index" Google Sheet, and when people start working on the next item (ADR-0008, at this time), they would add a description to row 8 in the sheet, reserving that number for their own usage.  Since gaps aren't a problem, this would avoid accidental duplicate numbered and annoying renumbering.